### PR TITLE
fix: lint warnings

### LINKED ISSUE #830: Remove unused constants,variables,args
Currently, you can see it at https://github.com/kodadot/nft-gallery/pull/828/files#file-src-app-vue-L18

image

### PR DESCRIPTION
**Pull request by GabrielBuragev**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #830
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=GyZf4szj5ZeC4WQza4CjZzsE1u27uJKL77QL4kH1JzGB6bg)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/rmrk/Profile/ProfileDetail.vue [hunk 1]
```diff
@@ -434,10 +434,6 @@ export default class Profile extends mixins(PrefixMixin, InfiniteScrollMixin) {
     this.legal = identityFields?.legal as string
   }
 
-  protected onCollectedChange(count, item) {
-    this.totalCollected = count
-  }
-
```
Comment (GabrielBuragev) [inline components/rmrk/Profile/ProfileDetail.vue:440]: Removed since it was not used anywhere in the codebase.

**Comment by petersopko** [comment]
@GabrielBuragev thanks for your contribution, after some further code reviewing, I find your PR quite incomplete? Idk what IDE are you using, but basically, you've removed couple of unused variables/functions/arguments, but my WebStorm is screaming at me with more similiar problems? Can you check / scan our codebase further and try to remove more of the unused stuff? 
image


btw: DeepSource could be your guide when it comes to reviewing your PR.

**Comment by petersopko** [comment]
@GabrielBuragev any update on this?

File: plugins/consola.ts [hunk 2]
```diff
@@ -7,6 +7,6 @@ declare module 'vue/types/vue' {
   }
 }
 
-export default ({ app }, inject) => {
+export default (ctx, inject) => {
```
Comment (vikiival) [inline plugins/consola.ts:10]:
```suggestion
export default (ctx, inject): void => {
```

**Review by vikiival — COMMENTED** [review]
Otherwise ok

File: components/series/utils.ts [hunk 3]
```diff
@@ -61,18 +61,18 @@ export const nftFn = (a: any): RowSeries => {
...
+// const sumFn = (acc: number, val: SimpleSeriesNFT) => acc + formatNumber(val)
 const soldFn = (acc: number, val: SimpleSeriesNFT) =>
   val.issuer !== val.currentOwner ? acc + 1 : acc
 const uniqueFn = (acc: Set<string>, val: SimpleSeriesNFT) =>
   acc.add(val.metadata)
-const onlyOwned = ({ issuer, currentOwner }: SimpleSeriesNFT) =>
-  issuer === currentOwner
+// const onlyOwned = ({ issuer, currentOwner }: SimpleSeriesNFT) =>
+//   issuer === currentOwner
 // const onlyBuyEvents = ({ events }: SimpleSeriesNFT) => events.filter((e: { interaction: string }) => e.interaction === 'BUY')
```
Comment (GabrielBuragev) [inline components/series/utils.ts:76]: Commented out since they were not used (averagePrice is not returned)

Comment (vikiival) [reply components/series/utils.ts:76]: 👀

**Review by petersopko — APPROVED** [review]

**Comment by petersopko** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 66.96 USD/KSM ~ 0.747 $KSM
🧗 [GyZf4szj5ZeC4WQza4CjZzsE1u27uJKL77QL4kH1JzGB6bg](https://kusama.subscan.io/account/GyZf4szj5ZeC4WQza4CjZzsE1u27uJKL77QL4kH1JzGB6bg)
🔗 [0x1439532d230c20a2be81fc135de220df628dd0579e9b755cb50a62bbe2a0945d](https://kusama.subscan.io/extrinsic/0x1439532d230c20a2be81fc135de220df628dd0579e9b755cb50a62bbe2a0945d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)
